Library/Homebrew/rubocops/components_redundancy.rb

```diff
@@ -37,9 +40,24 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
 
           stable_block = find_block(body_node, :stable)
           if stable_block
-            [:url, :sha256, :mirror].each do |method_name|
+            STABLE_BLOCK_METHODS.each do |method_name|
               problem "`#{method_name}` should be put inside `stable` block" if method_called?(body_node, method_name)
             end
+
+            unless stable_block.body.nil?
+              child_nodes = stable_block.body.begin_type? ? stable_block.body.child_nodes : [stable_block.body]
+              if child_nodes.all? { |n| n.send_type? && STABLE_BLOCK_METHODS.include?(n.method_name) }
+                problem "`stable do` should not be present with only #{STABLE_BLOCK_METHODS.join("/")}"
```

Comment:
Feels like there's a better way to word this but I don't have any suggestions. Maybe worth mentioning it's redundant rather than broken.